### ISSUE TITLE
Accessibility | Tabview ink bar accessibility problems

### ISSUE THREAD

**Issue by GiacomoDM** [primefaces/primeng#13902]
### Describe the bug

The TabView component correctly implements an `ul` list with a `lì` item for each tab. 
Inside the list, a `li` is appended at the end to render the ink bar that visually highlight the selected tab. This element should have `role="presentation"` and should be `aria-hidden`. 

![image](https://github.com/primefaces/primeng/assets/25552695/fb102222-8e28-4f78-87c0-0cdc9e1bf002)

PrimeVue corrent implementation: 
![image](https://github.com/primefaces/primeng/assets/25552695/df14d65d-4bc2-469a-baa4-922b56161d2d)
 

### Environment

Windows

### Reproducer

https://primeng.org/tabview

### Angular version

16.X

### PrimeNG version

16.5.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.16
